x-pack/plugins/console_extensions/spec/generated/xpack.rollup.get_jobs.json

```diff
@@ -0,0 +1,11 @@
+{
+  "xpack.rollup.get_jobs": {
+    "methods": [
+      "GET"
+    ],
+    "patterns": [
+      "_xpack/rollup/job/{id}",
+      "_xpack/rollup/job/"
```

Comment:
Hitting this endpoint without an id returns an exception, this conflicts with the documentation so is not a console bug, but an Elasticsearch bug (rollup team has been notified and will fix).